src/Raven.Server/Config/Categories/ServerConfiguration.cs

```diff
@@ -109,5 +109,15 @@ public class ServerConfiguration : ConfigurationCategory
         [DefaultValue(false)]
         [ConfigurationEntry("Server.LogsStream.Disable", ConfigurationEntryScope.ServerWideOnly)]
         public bool DisableLogsStream { get; set; }
+
+        [Description("When the number of minutes since the last time work was done on the database passes this threshold deep database cleanup will be done.")]
+        [DefaultValue(5)]
+        [ConfigurationEntry("Server.TimeFromLastWorkForDeepDatabaseCleanupInMinutes", ConfigurationEntryScope.ServerWideOnly)]
+        public double TimeFromLastWorkForDeepDatabaseCleanupInMinutes { get; set; }
+
+        [Description("When the number of minutes since the database last idle time passes this threshold regular database cleanup will be done.")]
+        [DefaultValue(10)]
+        [ConfigurationEntry("Server.TimeFromLastIdleTimeForRegularDatabaseCleanupInMinutes", ConfigurationEntryScope.ServerWideOnly)]
+        public double TimeFromLastIdleTimeForRegularDatabaseCleanupInMinutes { get; set; }
```

Comment:
Probably needs `Expert:` desgination.
@Danielle9897 Can you look over the actual text?

Reply:
For: **Server.TimeFromLastWorkForDeepDatabaseCleanupInMinutes**
`"EXPERT: A deep database cleanup will be done when this number of minutes has passed since the last time work was done on the database."`

For: **Server.TimeFromLastIdleTimeForRegularDatabaseCleanupInMinutes**
`"EXPERT: A regular database cleanup will be done when this number of minutes has passed since the last database idle time."
`

Reply:
@YonatanResh pls also add the units:
`[TimeUnit(TimeUnit.Minutes)]`

Reply:
done.